docs/python-quickstart.md

```diff
@@ -26,45 +26,431 @@ menu:
  -->
 
 
-# Python API Quickstart
+# Python CLI Quickstart
 
-## Installation
+Pyiceberg ships with a CLI that's available after installing the package.
 
-Iceberg python is currently in development, for development and testing purposes the best way to install the library is to perform the following steps:
+```sh
+➜  pyiceberg --help
+Usage: pyiceberg [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  --catalog TEXT
+  --verbose BOOLEAN
+  --output [text|json]
+  --uri TEXT
+  --credential TEXT
+  --help                Show this message and exit.
+
+Commands:
+  describe    Describes a namespace xor table
+  drop        Operations to drop a namespace or table
+  list        Lists tables or namespaces
+  location    Returns the location of the table
+  properties  Properties on tables/namespaces
+  rename      Renames a table
+  schema      Gets the schema of the table
+  spec        Returns the partition spec of the table
+  uuid        Returns the UUID of the table
+```
+
+Browsing the catalog
+
+```sh
+➜  pyiceberg --uri thrift://localhost:9083 list                       
+default
+nyc  
+```
+
+```sh
+➜  pyiceberg --uri thrift://localhost:9083 list nyc
+nyc.taxis
 ```
-git clone https://github.com/apache/iceberg.git
-cd iceberg/python
-pip install -e .
+
+```sh
+➜  pyiceberg --uri thrift://localhost:9083 list nyc
```

Comment:
What do you think about having a section early on for the config file so that we can avoid having `--uri` in every command?

Reply:
My initial idea was to have the user as quickly as possible up and running, but I changed the order. Let me know what you think.

Reply:
I agree with that, but it gets in the way. I'd probably start the intro with a simple example with the --uri and then move to config and more commands. That way, the user can skip config if they choose and continue to use --uri, but we don't have to put it in every command (and update it later).